cmake/hardening.cmake

```diff
@@ -1,9 +1,7 @@
 # Depends on os.cmake and profile.cmake modules.
-# Uses `ENABLE_FUZZER` option and `TARGET_OS_FREEBSD` variable.
+# Uses `ENABLE_FUZZER` option and `TARGET_OS_DARWIN` variable.
 
-# LuaJIT in FreeBSD doesn't work with PIC (gh-7640),
```

Comment:
Isn't #7640 still relevant?

Reply:
https://github.com/tarantool/tarantool/pull/8784/commits/376b0b093105fe0cd6232a34d1e2d6b92980c26c

**build: do not disable hardening on FreeBSD**
> It was disabled because dlsym tests failed when PIC is on (https://github.com/tarantool/tarantool/issues/7640).
> However, later it turned out that the issue is not related to PIC, and
> LuaJIT was turned off for such tests by commit https://github.com/tarantool/tarantool/commit/67e79b15f03fb6b7149a8f8a6f35932d785a314c ("test: turn
> LuaJIT off in tests with dlsym"). Now it's safe to turn on PIC for FreeBSD.

Reply:
Thanks for the explanation!